Sources/KVKCalendar/Timeline+Extension.swift

```diff
@@ -320,10 +320,10 @@ extension TimelineView {
         timeLabels.first(where: { $0.hashTime == hour })
     }
     
-    func createTimesLabel(start: Int) -> (times: [TimelineLabel], items: [UILabel]) {
+    func createTimesLabel(start: Int, end: Int) -> (times: [TimelineLabel], items: [UILabel]) {
         var times = [TimelineLabel]()
         var otherTimes = [UILabel]()
-        for (idx, txtHour) in timeSystem.hours.enumerated() where idx >= start {
+        for (idx, txtHour) in timeSystem.hours.enumerated() where idx >= start && idx <= end {
```

Comment:
Why do we need this?

Reply:
Here's what I need: museum staff to manage their schedules during the workday.The museum has opening hours, and times outside the range of opening hours are not displayed.So the end time is needed and the labels that are later than the end time are not displayed.This behavior is the same as the start time.